packages/cli/plugin-bff/src/cli.ts

```diff
@@ -16,7 +16,7 @@ import { registerModernRuntimePath } from './helper';
 const DEFAULT_API_PREFIX = '/api';
 const TS_CONFIG_FILENAME = 'tsconfig.json';
 
-export default (): CliPlugin<AppTools> => ({
+export default (): CliPlugin<AppTools<'webpack'>> => ({
```

Comment:
'webpack' should be a default value, users don't need to explicitly declare it.

Reply:
The AppTools CliPlugin has 3 types temporarily: `AppTools<'webpack'>` 、 `AppTools<'rspack'>` 、 `AppTools<undefined>`. The default type is `AppTools<undefined>`, and the CliPlugin can use SharedTypes in defines. This is means the CliPlugin not only use in the base of webpack but alse use in rspack base

Reply:
It sounds like a breaking change for Plugin developers.

Reply:
Or we can make the `CliPlugin<'webpack'>` default? After all, `Rspack` is in experimental status. 

But with the development of the `Rspack`, It's  necessary to distinguish between 'webpack' Plugin and 'rspack' Plugin. 
When user is not specify the CliPlugin type, is should be shared CliPlugin.

Reply:
Yes, we should make the `CliPlugin<'webpack'>` as a default value.

Reply:
done